integr8ly/installation
# Intly 1177

## Verification
- browse to: https://master.pbrookes-cf77.openshiftworkshop.com
- login as customer-admin
- only see fuse namespace.
